.github/ISSUE_TEMPLATE/bug-in-sample.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug in Sample
+about: Report a bug in an existing sample project
+title: '[BUG]'
+labels: 'bug'
+assignees: ''
+
+---
+## Which sample is this report about?
+- Sample name or link: 
+
+## Describe the bug
+<!-- Please add a clear and concise description of the problem below. -->
+
+## Steps to reproduce
+Steps to reproduce the behavior:
+1. First step 
+2. ..
+3. ..
+
+## Expected behavior
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Smartphone (please complete the following information):
+ - Device: 
+ - OS: 
+ - Version:
```

Comment:
IMO somewhere we should require the Bitmovin Player SDK version

Reply:
Thought about that, but unless explicitly changed the samples are always on the latest version. So I decided against it for now.

Reply:
Yes I know, but so easy to change and would save us a lot of effort.
Maybe we could just pre-fill it with value: `latest`. Then you can change it if it is about older versions

Reply:
K, I'll take that. Better having it and not needing rather than needing and not having it 😉

Reply:
https://github.com/bitmovin/bitmovin-player-ios-samples/pull/34/commits/bb868ced05abd20c16f8e5d4ab634f918e4bad97

Reply:
no default value? Still fine like this

Reply:
I'd prefer an actual version number here (thought about it)